Mathlib/RingTheory/PowerSeries/WellKnown.lean

```diff
@@ -64,6 +65,52 @@ theorem map_invUnitsSub (f : R →+* S) (u : Rˣ) :
 
 #align power_series.map_inv_units_sub PowerSeries.map_invUnitsSub
 
+/-- Give `d : ℕ`. The power series for `1 / ((1 - x) ^ (d + 1))`. -/
+def invOneSubPow (d : ℕ) : PowerSeries R :=
+  mk fun n => Nat.choose (d + n) d
+
+theorem invOneSubPow_zero_eq_invUnitsSub_one : invOneSubPow 0 = invUnitsSub (1 : Rˣ) := by
+  rw [invOneSubPow, invUnitsSub]
+  simp only [zero_add, Nat.choose_zero_right, Nat.cast_one, one_pow, divp_one]
+
+theorem one_sub_inv_eq_invOneSubPow_zero {k : Type _} [Field k] :
+    (1 - X : PowerSeries k)⁻¹ = invOneSubPow 0 := Eq.symm <| by
```

Comment:
Why are we assuming `k` is a field here, the constant coefficient of `(1 - X)` is invertible, so `1 - X` as a power series is invertible in a commutative ring already. We have this in mathlib as well.

If possible, can you remove the `field` assumption whereever possible